.ruby-version

```diff
@@ -1 +1 @@
-2.7.6
+3.0.6
```

Comment:
make sure that is available on the samson hosts ... last 2.7.8 upgrade failed on that

Reply:
We added a bunch of mission versions last week; good call-out, though.